### PR TITLE
option to not crop in dataset run

### PR DESCRIPTION
**Pull request by ssarkar2**
# What does this PR do?
Currently for dataset run, the dataset will be cropped to fixed len. Add option to not crop

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Review by libinta — APPROVED** [review]

File: examples/text-generation/run_generation.py [hunk 1]
```diff
@@ -544,7 +546,7 @@ def generate():
         raw_dataset = raw_dataset.remove_columns([name for name in raw_dataset.column_names if name != column_name])
 
         # Set the prompt length to args.max_input_tokens if > 0 else default to 16
```
Comment (regisss) [inline examples/text-generation/run_generation.py:548]: Can you also update this comment for the -1 case?

**Review by regisss — COMMENTED** [review]
I just left a very minor comment

File: examples/text-generation/run_generation.py (hunk 1 above)
Comment (ssarkar2) [reply examples/text-generation/run_generation.py]: thanks, fixed

**Review by regisss — APPROVED** [review]
LGTM!

**Pull request merged**